roles/ubuntu-basic/tasks/main.yml

```diff
@@ -10,6 +10,7 @@
       - htop
       - mc
       - ncdu
+      - snapd
```

Comment:
Man könnte das auch in die slack rolle als exta install machen, dann kann man die rolle auch wo anders benutzen

Reply:
gute Idee, dann wäre die Rolle selbstständiger